com.unity.netcode.gameobjects/Components/HalfVector4.cs

```diff
@@ -0,0 +1,68 @@
+using UnityEngine;
+
+namespace Unity.Netcode.Components
+{
+    internal struct HalfVector4 : INetworkSerializable
+    {
+        public ushort X;
+        public ushort Y;
+        public ushort Z;
+        public ushort W;
+
+        public void NetworkSerialize<T>(BufferSerializer<T> serializer) where T : IReaderWriter
+        {
+            serializer.SerializeValue(ref X);
```

Comment:
Same comment as HalfVector3 - pack/unpack

Reply:
Same as previous response about the ushort values being half floats